src/renderer/src/components/Sidebar/index.tsx

```diff
@@ -28,7 +28,7 @@ export const Sidebar = ({ className, ...props }: TProps) => {
   const { logout } = useLogin()
 
   return (
-    <aside className={StyleHelper.mergeStyles('flex h-full w-16 min-w-16 flex-col bg-gray-800', className)} {...props}>
+    <aside className={StyleHelper.mergeStyles('flex h-full min-w-16 flex-col bg-gray-800', className)} {...props}>
```

Comment:
Use:
```suggestion
    <aside className={StyleHelper.mergeStyles('flex h-full w-16 max-w-16 min-w-16 flex-col bg-gray-800', className)} {...props}>
```

Translate "Configurações" to "Config.". What do you think?